sources/documents/2_RTB/PianoDiProgetto/Preventivo/preventivo.tex

```diff
@@ -20,4 +20,5 @@ \section{Preventivo}
 
 %\subsection{RTB}
 %TODO
-\input{Preventivo/Sprint/sprint-1}
\ No newline at end of file
+\input{Preventivo/Sprint/sprint-1}
+\input{Preventivo/Sprint/sprint-2}
```

Comment:
Credo sia da eliminare l'input dello sprint 2 in quanto non esistente come file al momento della compilazione